vmconf.py

```diff
@@ -13,4 +13,47 @@
 # limitations under the License.
 
 """Stores various configuration options and constants for Oppia-ml."""
-pass
+
+# The platform for the storage backend. This is used in the model-switching
+# code in core/platform.
+PLATFORM = 'gce'
+
+# Whether we should serve the development or production experience.
+# DEV_MODE should only be changed to False in the production environment.
+DEV_MODE = True
+
+# Default communication URL to be used to communicate with Oppia server.
+# This URL will be used in local development.
+DEFAULT_COMMUNICATION_URL = 'https://localhost'
+
+# Default communication PORT to be used to communicate with Oppia server.
+# This URL will be used in local development.
+DEFAULT_COMMUNICATION_PORT = 8181
+
+# Communication URL which is to be used in production environment to
+# communicate with Oppia server.
+SERVER_COMMUNICATION_URL = 'https://www.oppia.org'
+
+# Communication PORT which is to be used in production environment to
+# communicate with Oppia server.
+SERVER_COMMUNICATION_PORT = 80
+
+# Default ID which is to be used in local development environment.
+DEFAULT_VM_ID = 'vm_default'
+
+# Default shared key which is to be used in local development environment.
+DEFAULT_VM_SHARED_SECRET = '1a2b3c4e'
+
+# Name of metadata parameter which will contain ID of the VM in production
+# environment in GCE.
+METADATA_VM_ID_PARAM_NAME = 'vm_id'
```

Comment:
Oppia needs to know about them because we have different shared key for each VM. Oppia stores vm_id <=> shared_key mapping. So Oppia needs to know which VM it is communicating so that it can verify the signature.